test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetObjectMonitorUsage/objmonusage001.java

```diff
@@ -61,8 +61,15 @@ public static int run(String argv[], PrintStream out) {
             syncObject[i] = new Object();
             runn[i] = new objmonusage001a(mainThread, i, syncObject[i]);
         }
+        // Virtual threads are not supported by the GetObjectMonitorUsage. Correct
+        // the expected values if the test is executed with MainWrapper=virtual.
```

Comment:
"MainWrapper" is not the proper terminology any more.  It's "Test Thread Factory" (JTREG_TEST_THREAD_FACTORY=Virtual).

Reply:
Good suggestion, thanks. Then I'd suggest this:
```
        // Virtual threads are not supported by the GetObjectMonitorUsage.
        // Correct the expected values if the test is executed with the
        // JTREG_TEST_THREAD_FACTORY=Virtual.
```

Reply:
You can drop "the" from "with the JTREG_TEST_THREAD_FACTORY=Virtual"

Reply:
And drop "the" from "the GetObjectMonitorUsage".

Reply:
Thank you. Updated.